siriuspy/siriuspy/devices/blctrl.py

```diff
@@ -0,0 +1,85 @@
+"""Beamline Control."""
+
+from .device import Device as _Device
+
+
+class BLCtrl(_Device):
```

Comment:
maybe we could use a name that refers to PPS system, this name is a little bit generic

Reply:
I am not sure what else in the CAX will be useful and implemented in this device, that is why I left generic. Another option is to break into many classes.

Reply:
I like this last approach. Or at least include the prefix `pps_` in the property names.

Reply:
I also prefer the last approach

Reply:
class renamed to `BLPPSCtrl`